calico-cloud/networking/configuring/dual-tor.mdx

```diff
@@ -646,6 +646,16 @@ as their source and destination. For example:
 
 The only connections using interface-specific addresses should be BGP.
 
+:::note
+
+- Some Calico features, such as Egress Gateways, will not work correctly if per-interface
+  IP addresses are used instead of a node's stable IP address.
+- If interface-specific addresses are seen on connections where the stable IP address is
+  expected, the {{nodecontainer}} IP auto-detection method must be adjusted to detect the stable IP.
+  This can be configured via the {{prodname}} [Installation resource](/calico-cloud/reference/installation/api#operator.tigera.io/v1.NodeAddressAutodetection).
```

Comment:
```suggestion
  This can be configured via the {{prodname}} [Installation resource](/calico-cloud/reference/installation/api#operator.tigera.io/v1.NodeAddressAutodetection).
```